src/aleph/vm/sevclient.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+from pathlib import Path
+
+
+class SevClient:
+    def __init__(self, sev_dir: Path):
+        self.sev_dir = sev_dir
+        self.certificates_dir = sev_dir / "platform"
+        self.certificates_dir.mkdir(exist_ok=True, parents=True)
+        self.certificates_archive = self.certificates_dir / "certs_export.cert"
+
+    def sevctl_cmd(self, *args) -> subprocess.CompletedProcess:
+        result = subprocess.run(
```

Comment:
Using the `subprocess` module is synchronous and would block the entire event loop.

[asyncio.subprocess](https://docs.python.org/3/library/asyncio-subprocess.html#asyncio-subprocess) should be used instead ;-)